Mage/src/main/java/mage/constants/ComparisonType.java

```diff
@@ -4,35 +4,23 @@
  * Created by IGOUDT on 5-3-2017.
  */
 public enum ComparisonType {
-    FEWER_THAN("<", "fewer", "than"),
-    FEWER_THAN_OR_EQUAL_TO("<=", "fewer", "than or equal to"),
```

Comment:
Paper's rules naming should be a priority:
 * greater than or equal to
 * less than or equal to

![shot_230707_125627](https://github.com/magefree/mage/assets/8344157/aaede9d7-d166-4925-8c3c-94f6d155d150)

Reply:
But there are multiple wordings in paper card text:

https://scryfall.com/search?q=o%3A%22than%20or%20equal%22

https://scryfall.com/search?q=o%3A%22or%20less%22%20or%20o%3A%22or%20greater%22

Hence my suggestion.

Reply:
Maybe we could have cases for both wording styles - they would do the same thing but they would have different text.

Reply:
I suppose we could, but I'm pretty sure that would be more confusing overall to have two equivalent cases. Effects/conditions can set the appropriate card text from a single case.

Reply:
What do you think I should do?

Reply:
Personally I'm happy with how it is now.